modules/5_install/install.tf

```diff
@@ -400,7 +401,7 @@ resource "null_resource" "powervs_config" {
 
 resource "null_resource" "upgrade" {
   depends_on = [null_resource.install, null_resource.powervs_config]
-  count      = var.upgrade_version != "" ? 1 : 0
+  count      = var.upgrade_version != "" || var.upgrade_image != "" ? 1 : 0
   triggers = {
     upgrade_version = var.upgrade_version
```

Comment:
Don't we need to trigger this module on image change as well? @varad-ahirwadkar

Reply:
Hi @Prajyot-Parab, I have added `var.upgrade_version`  to trigger.